migrations/Version202107121456123210_taoDacSimple.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
```

Comment:
i don't remember if we are adding the license inside the migrations

Reply:
migrations are generated by script. I'm not interfere with this file template